app/_common/api/instance.ts

```diff
@@ -29,11 +29,11 @@ instance.interceptors.response.use(
   },
   async error => {
     const { status, config } = error.response;
-    const refreshToken = localStorage.getItem("refreshToken");
+    const refreshToken = getCookie("refreshToken", "");
```

Comment:
`await` 키워드를 안붙여주면 값이 없는 상태에서 요청이 들어가는 것 같아요. 

```suggestion
    const refreshToken = await getCookie("refreshToken", "");
```

![image](https://github.com/Open-Eye-Im-Developer/MoGakGo-FE/assets/86952779/86f9d522-103f-480a-a2e6-b33ea138cb3b)